src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderStateController.php

```diff
@@ -153,6 +169,14 @@ function (array $language) {
      */
     public function editAction(int $orderStateId, Request $request)
     {
+        if ($this->get('prestashop.adapter.multistore_feature')->isActive()) {
+            $this->addFlash(
+                'info',
+                $this->trans('Note that this feature is available in all shops context only. It will be added to all your stores.',
+                    'Admin.Notifications.Info')
```

Comment:
```suggestion
                $this->trans(
                    'Note that this feature is available in all shops context only. It will be added to all your stores.',
                    'Admin.Notifications.Info'
                )
```